tests/config/baseTest.ts

```diff
@@ -36,6 +40,7 @@ type BaseOptions = {
 };
 type BaseFixtures = {
   platform: 'win32' | 'darwin' | 'linux';
+  getPlaywright: (options?: PlaywrightModeSetupOptions) => Promise<typeof import('../../index')>;
```

Comment:
Just add an option to BaseOptions called `service`, and use it in `playwright` fixture. Later, `test.use({ service: whatever })` in the test file.

This will make a nice prototype for the actual use.

Reply:
What about this one? I still think we either do the `service` option or revert any changes in this file.